spec/observables/dom/ajax-spec.ts

```diff
@@ -1366,6 +1366,173 @@ describe('ajax', () => {
 x-custom-header: test
 x-headers-are-fun: <whatever/> {"weird": "things"}`);
   });
+
+  describe('with params', () => {
+    it('should allow passing of search params as a dictionary', () => {
+      ajax({
+        method: 'GET',
+        url: '/whatever',
+        params: { foo: 'bar', whatever: '123' },
+      }).subscribe();
+
+      const mockXHR = MockXMLHttpRequest.mostRecent;
+
+      mockXHR.respondWith({
+        status: 200,
+        responseText: JSON.stringify({ whatever: 'I want' }),
+      });
+
+      expect(mockXHR.url).to.equal('/whatever?foo=bar&whatever=123');
+    });
+
+    it('should allow passing of search params as an entries array', () => {
+      ajax({
+        method: 'GET',
+        url: '/whatever',
+        params: [
+          ['foo', 'bar'],
+          ['whatever', '123'],
+        ],
+      }).subscribe();
+
+      const mockXHR = MockXMLHttpRequest.mostRecent;
+
+      mockXHR.respondWith({
+        status: 200,
+        responseText: JSON.stringify({ whatever: 'I want' }),
+      });
+
+      expect(mockXHR.url).to.equal('/whatever?foo=bar&whatever=123');
+    });
+
+    it('should allow passing of search params as a string', () => {
+      ajax({
+        method: 'GET',
+        url: '/whatever',
+        params: '?foo=bar&whatever=123',
+      }).subscribe();
+
+      const mockXHR = MockXMLHttpRequest.mostRecent;
+
+      mockXHR.respondWith({
+        status: 200,
+        responseText: JSON.stringify({ whatever: 'I want' }),
+      });
+
+      expect(mockXHR.url).to.equal('/whatever?foo=bar&whatever=123');
+    });
+
+    it('should allow passing of search params as a URLSearchParams object', () => {
+      const params = new URLSearchParams();
+      params.set('foo', 'bar');
+      params.set('whatever', '123');
+      ajax({
+        method: 'GET',
+        url: '/whatever',
+        params: '?foo=bar&whatever=123',
+      }).subscribe();
```

Comment:
issue: AFAICT, this isn't passing the `params` object; it's passing a string.

Reply:
🤦‍♂️